### ISSUE TITLE
Grafana install error

### ISSUE THREAD

**Issue by ricardoseriani** [laravel/homestead#1167]
## Versions
  - Vagrant: 2.2.4
  - Provider: Virtualbox 6.0.8
  - Homestead: 7.2.1
  - Homestead git: version_9

## Host operating system
Ubuntu 16.04 - Xenial

## Homestead.yaml
```yaml
---
# box: laravel/homestead-7
ip: "192.168.10.10"
memory: 2048
cpus: 2
provider: virtualbox
version: ">=0.5.0"

authorize: ~/.ssh/id_rsa.pub

keys:
    - ~/.ssh/id_rsa

folders:
    - map: ~/sites
      to: /home/vagrant/sites

sites:
    - map: cobaia.test
      to: /home/vagrant/sites/cobaia

databases:
    - cobaia

variables:
    - key: 'APP_ENV'
      value: 'development'

ports:
    - send: 63790
      to: 6379

features:
    - grafana: true

```

## Vagrant destroy & up output
https://gist.github.com/ricardoseriani/b4393555d1db5abcfd81b9c49ca19437

## Expected behavior
Install grafana.

## Actual behavior
Doesn't install grafana due to `GPG error: https://packagecloud.io/grafana/stable/debian stretch InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 40F370A1F9081B64`

## Steps to reproduce
1. vagrant destroy
2. vagrant up

**Comment by ricardoseriani** [comment]
I can work on this, but since it's base box is being updated to ubuntu 19.04, I think that this should be hold until the update is finished.

**Comment by stefnats** [comment]
The packagecloud.io repository is deprecated and they recommend using this installation:
https://grafana.com/docs/installation/debian/

**Comment by ricardoseriani** [comment]
Since the ubuntu version will remain the same (#1144), I'm working on this fix and will make a PR.

**Comment by svpernova09** [comment]
Resolved on master via e913e0cf4a1e8505fed6b14d085c4b2cecca758f

**Comment by svpernova09** [comment]
Fixed for v8 in v8.5.4 and on `version_9`
